manishpandey.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "learnman18",
+    "email": "manishvirat18.mp@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://www.manishpandey.is-a.dev/"
```

Comment:
eh, you dont want to make your website recursive, right?

`manishpandey.is-a.dev` (this json file) CNAME'd to `manishpandey.is-a.dev` (this file again)
And it doesn't display anything

Reply:
Or if you plan to add the `www` sub-subdomain, add a separate `www.manishpandey.json` and add the content onto it. Like, github pages or smth